manifest.json

```diff
@@ -7,8 +7,14 @@
   "content_scripts": [
     {
       "matches": ["<all_urls>"],
-      "run_at": "document_start",
-      "js": ["content/content.js"]
+      "js": ["content/content.js"],
+      "run_at": "document_start"
     }
-  ]
+  ],
+  "browser_specific_settings": {
+    "gecko": {
+      "id": "@forem",
+        "strict_min_version": "42.0"
```

Comment:
We can probably bump this up. The Firefox stable release is currently at 79. Maybe set this to `60.0` or `70.0`?